app/server/index.py

```diff
@@ -31,6 +31,8 @@ def do_GET(self):
                     shell = True,
                     stdout = subprocess.PIPE)
                 data = output.communicate()[0]
+                ##替换data中的空格和反斜杠
```

Comment:
Please update this to an English comment.

Reply:
Remove single charactor "\" or single blank space.